<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addHabit</class>
 <widget class="QFrame" name="addHabit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Habit</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/normalWindowsUser/Desktop/Klotman.png</normaloff>C:/Users/normalWindowsUser/Desktop/Klotman.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <widget class="QLabel" name="window">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/addHabitWindow.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(71, 71, 71);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/AddHabitButton.png</normaloff>:/res/AddHabitButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(71, 71, 71);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/cancelHabitButton.png</normaloff>:/res/cancelHabitButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="closeB">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/close.png</normaloff>:/res/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="minimizeB">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/minimize.png</normaloff>:/res/minimize.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/version.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/addHabitTitle.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>150</y>
     <width>58</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 12pt &quot;Century Gothic&quot;;

</string>
   </property>
   <property name="text">
    <string>Name *</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameIn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>315</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 12pt &quot;Century Gothic&quot;;

</string>
   </property>
   <property name="text">
    <string>Question *</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="questionIn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>313</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;

</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionIn">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>290</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>87</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 12pt &quot;Century Gothic&quot;;

</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>103</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 12pt &quot;Century Gothic&quot;;


</string>
   </property>
   <property name="text">
    <string>Grace (Days)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="graceSpinner">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>360</y>
     <width>44</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;

</string>
   </property>
  </widget>
  <zorder>window</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>nameIn</zorder>
  <zorder>nameIn</zorder>
  <zorder>label</zorder>
  <zorder>descriptionIn</zorder>
  <zorder>label_3</zorder>
  <zorder>graceSpinner</zorder>
  <zorder>label_5</zorder>
  <zorder>addButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>closeB</zorder>
  <zorder>minimizeB</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label</zorder>
  <zorder>nameIn</zorder>
  <zorder></zorder>
  <zorder>label_5</zorder>
  <zorder>graceSpinner</zorder>
  <zorder>label</zorder>
  <zorder>nameIn</zorder>
  <zorder>label_2</zorder>
  <zorder>questionIn</zorder>
  <zorder>descriptionIn</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>graceSpinner</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
